doc/sphinx/language/cic.rst

```diff
@@ -721,7 +721,8 @@ called the *context of parameters*. Furthermore, we must have that
 each :math:`T` in :math:`(t:T)∈Γ_I` can be written as: :math:`∀Γ_P,∀Γ_{\mathit{Arr}(t)}, S` where
 :math:`Γ_{\mathit{Arr}(t)}` is called the *Arity* of the inductive type t and :math:`S` is called
 the sort of the inductive type t (not to be confused with :math:`\Sort` which is the set of sorts).
-** Examples** The declaration for parameterized lists is:
+
+**Examples** The declaration for parameterized lists is:
```

Comment:
It's probably even better to use the `.. example::` environment there. It was sometimes forgotten during the migration.

Reply:
Alright, I added commits for that which should be squashed before merging; or do you want me to create a separate commit that replaces the bold "Example" text with the `example` directive?